ci-operator/step-registry/ipi/conf/aws/usage-info/ipi-conf-aws-usage-info-ref.metadata.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@jianlinliu I think for JSON files, the indentation is 4 spaces, not 8 as what's there currently.

Reply:
The file is automatically generated by prow make command.